### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by Au-LiuJY** [riolys/DisDiff#2]
After using SimAC timestep search , when it comes to"
---   using CAE loss   ---
---   using MSS   ---
Step #0, loss: 0.2744, prior_loss: 0.0813, instance_loss: 0.1930
Step #1, loss: nan, prior_loss: nan, instance_loss: nan
Step #2, loss: nan, prior_loss: nan, instance_loss: nan
PGD loss - step 0, loss: nan
PGD loss - step 1, loss: nan
PGD loss - step 2, loss: nan
PGD loss - step 3, loss: nan
PGD loss - step 4, loss: nan
PGD loss - step 5, loss: nan
Step #0, loss: nan, prior_loss: nan, instance_loss: nan
Step #1, loss: nan, prior_loss: nan, instance_loss: nan
Step #2, loss: nan, prior_loss: nan, instance_loss: nan
Step #0, loss: nan, prior_loss: nan, instance_loss: nan
Step #1, loss: nan, prior_loss: nan, instance_loss: nan
Step #2, loss: nan, prior_loss: nan, instance_loss: nan
PGD loss - step 0, loss: nan
PGD loss - step 1, loss: nan
PGD loss - step 2, loss: nan
PGD loss - step 3, loss: nan
PGD loss - step 4, loss: nan
PGD loss - step 5, loss: nan
Step #0, loss: nan, prior_loss: nan, instance_loss: nan
Step #1, loss: nan, prior_loss: nan, instance_loss: nan
Step #2, loss: nan, prior_loss: nan, instance_loss: nan
Step #0, loss: nan, prior_loss: nan, instance_loss: nan
Step #1, loss: nan, prior_loss: nan, instance_loss: nan
Step #2, loss: nan, prior_loss: nan, instance_loss: nan
PGD loss - step 0, loss: nan
PGD loss - step 1, loss: nan
PGD loss - step 2, loss: nan
PGD loss - step 3, loss: nan
PGD loss - step 4, loss: nan
PGD loss - step 5, loss: nan
"
After "Step #0, loss: 0.2744, prior_loss: 0.0813, instance_loss: 0.1930",all the loss will be nan.
Hoping that you can answer my question at your convenience.

**Comment by Au-LiuJY** [comment]
> Sorry, I haven't met this question. I wonder if the same question exists when you proceed with the Simac source code. By the way, I don't think Simac's method makes sense since its code has something wrong. For example, in line 700 of time_feature.py, loss = loss + target_loss.detach().item(), the target_loss is not used for backpropagation. The step selection is also different from the paper since the selection step is before training in the source code.

Hi,dear author.I just simply use bash scripts/disdiff.sh to run your demo.When I process the SimAC soure code,the result seems like:---   using SimAC timestep search   ---
PGD loss - step 0, index : 1, loss: 0.35028544068336487, score: 2701037.5, t : tensor([214]), ts_len: 1000
PGD loss - step 0, index : 2, loss: 0.3946853280067444, score: 3380897280.0, t : tensor([181]), ts_len: 1000
PGD loss - step 0, index : 3, loss: 0.003192029893398285, score: 88597304.0, t : tensor([975]), ts_len: 1000
PGD loss - step 0, index : 4, loss: 0.081753671169281, score: 931493376.0, t : tensor([542]), ts_len: 1000
PGD loss - step 0, index : 5, loss: 0.00803731195628643, score: 191570832.0, t : tensor([868]), ts_len: 1000
del_t 214 max_t 181
PGD loss - step 1, index : 1, loss: 0.0088163698092103, score: 156367936.0, t : tensor([870]), ts_len: 960
PGD loss - step 1, index : 2, loss: 0.5954245328903198, score: 3375317504.0, t : tensor([74]), ts_len: 960
PGD loss - step 1, index : 3, loss: 0.2823884189128876, score: 1842061184.0, t : tensor([273]), ts_len: 960
PGD loss - step 1, index : 4, loss: 0.38245290517807007, score: 2421717504.0, t : tensor([187]), ts_len: 960
PGD loss - step 1, index : 5, loss: 0.07512351870536804, score: 727062272.0, t : tensor([558]), ts_len: 960
del_t 870 max_t 74
…………

**Comment by Au-LiuJY** [comment]
I just simply change the BASE_DIR and MODEL_PATH

![Image](https://github.com/user-attachments/assets/060f039b-7f63-4d92-8a4c-7dcb2a85c31d)
